<?xml version="1.0" encoding="UTF-8"?>
<testsuites name="Mocha Tests" time="35.111" tests="1" failures="1">
  <testsuite name="Root Suite" timestamp="2023-02-08T19:03:59" tests="0" file="cypress\e2e\user-management\create-new-user.cy.js" time="0.000" failures="0">
  </testsuite>
  <testsuite name="Create New User in User management" timestamp="2023-02-08T19:03:59" tests="1" time="35.096" failures="1">
    <testcase name="Create New User in User management Navigate to User Management and Create new user" time="0.000" classname="Navigate to User Management and Create new user">
      <failure message="Timed out retrying after 12000ms: Expected to find content: &apos;FEDEX&apos; within the element: [ &lt;li&gt;, 143 more... ] but never did." type="AssertionError"><![CDATA[AssertionError: Timed out retrying after 12000ms: Expected to find content: 'FEDEX' within the element: [ <li>, 143 more... ] but never did.
    at Context.eval (webpack:///./cypress/support/commands.js:336:14)]]></failure>
    </testcase>
  </testsuite>
</testsuites>